src/CHANGES.txt

```diff
@@ -62,6 +62,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Accommodate VS 2017 Express - it's got a more liberal license then VS
       Community, so some people prefer it (from 2019, no more Express)
     - vswhere call should also now work even if programs aren't on the C: drive.
+    - Script/Main.py now uses importlib instead of imp module.
```

Comment:
Might be worth adding motivation.. imp deprecated and will be removed in Py#.#?
Are we still using imp elsewhere?